OrbitGl/App.cpp

```diff
@@ -1049,6 +1049,29 @@ std::shared_ptr<Process> OrbitApp::FindProcessByPid(uint32_t pid) {
 
 void OrbitApp::UpdateProcess(const std::shared_ptr<Process>& process) {
   absl::MutexLock lock(&process_map_mutex_);
+  // We want to move function map to new process object
+  auto it = process_map_.find(process->GetID());
```

Comment:
Maybe rename this as well, maybe to `old_process_it`?

Reply:
done